Update Unity troubleshooting (#4522)
getsentry/sentry-docs: src/platforms/unity/troubleshooting.mdx

## Installation Issues

> ### Unable to add package Sentry Unity Package from Git

Reviewer: Two points down the list we already cover this issue. But maybe we need to highlight the actual error message in a better way.

Reviewer 2: Suggest moving the text sample and the screenshot under the existing content and then update that text to include all of the scenarios (trailing whitespace, additional spaces or lines).

Author: I moved the Sample and the Screenshot under the existing content.
About the additional scenarios, they may not be necessary since Unity converts newlines to whitespace.

Author: Additionally, since if you add whitespace can also result in the same error, I altered the line `Confirm the URL doesn't have a trailing whitespace at the end` to
`Confirm the URL doesn't have a trailing whitespace between the URL`. Is that ok for you @imatwawana and @bitsandfoxes?

### Package Manager - `Cannot perform upm operation: Unable to add package`

If you encounter the following error:

```
[Package Manager Window] Unable to add package [https://github.com/getsentry/unity.git#0.9.1 ]:
  Error when executing git command. fatal: invalid refspec '0.9.1 '
```

> Confirm the URL doesn't have a trailing whitespace between the URL. The Unity Package Manager will fail to find the package if a trailing whitespace is appended.

Reviewer 3: Between?

Reviewer 2: In this case just "in" is correct rather than "between". You would use "between" if there are two things (e.g., spaces between the beginning and end of the URL).
 
```suggestion
Confirm the URL doesn't have a trailing whitespace or spaces in the URL. The Unity Package Manager will fail to find the package if a trailing whitespace is appended.
```

> Confirm the URL doesn't have a trailing whitespace or spaces in the URL. The Unity Package Manager will fail to find the package if a trailing whitespace is appended.

Reviewer 2: Just realized the second sentence doesn't match the first one now. 

```suggestion
Confirm the URL doesn't have a trailing whitespace or spaces in the URL. The Unity Package Manager will fail to find the package if whitespace is appended.
```

This can happen if you've copied some of the SDK files directly to the Assets folder and the `Sentry.Unity.Editor.iOS.dll` which runs when you build an iOS player wasn't able to copy the `Sentry.framework` to the final app. Enabling debug mode in the Sentry editor window would display more information in the Unity console.

You can resolve this issue by installing the SDK with UPM.

> ## Events not sent

Reviewer 2: ```suggestion
## Events Not Sent
```

### Universal Windows Platform

> In order to send events to Sentry, you will need to activate the InternetClient Capability on your Player Settings.

Reviewer 2: ```suggestion
In order to send events to Sentry, you will need to activate the InternetClient Capability in your Player Settings.
```
